docs/notebooks/how_to_build_and_optimize_qaoa_using_swap_strategy.ipynb

```diff
@@ -0,0 +1,742 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import json\n",
+    "import networkx as nx\n",
+    "\n",
+    "seed=100\n",
+    "graph = nx.random_regular_graph(3, 10, seed=seed)\n",
+    "nx.draw(graph, with_labels=True, node_size=1000, node_color=\"deepskyblue\", pos=nx.spring_layout(graph, seed=seed))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This graph has 10 nodes and 15 edges, and it can be encoded in 10 qubits."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "10\n"
+     ]
+    }
+   ],
+   "source": [
+    "num_qubits = len(graph.nodes)\n",
+    "print(num_qubits)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The graph utils included in the `qopt-best-practice` library allow you to easily switch between the max-cut graph and cost hamiltonian."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "[('IIIIIIIIZZ', 1.0), ('IIIIIIZIIZ', 1.0), ('IIIIIIIZIZ', 1.0), ('IIIIZIIIZI', 1.0), ('IIZIIIIIZI', 1.0), ('IIZIIIIZII', 1.0), ('IIIIIIZZII', 1.0), ('ZIZIIIIIII', 1.0), ('IIIIZZIIII', 1.0), ('IIIIZIZIII', 1.0), ('IZIZIIIIII', 1.0), ('IIIZIZIIII', 1.0), ('ZIIZIIIIII', 1.0), ('ZZIIIIIIII', 1.0), ('IZIIIZIIII', 1.0)]\n"
+     ]
+    }
+   ],
+   "source": [
+    "from qopt_best_practices.utils import build_max_cut_paulis, build_max_cut_graph\n",
+    "\n",
+    "cost_hamiltonian = build_max_cut_paulis(graph)\n",
```

Comment:
I would rename the variable `cost_hamiltonian` to `max_cut_paulis`, to use the same notation as the `qopt_best_practices` package